sources/packages/web/src/services/ApplicationService.ts

```diff
@@ -39,6 +40,12 @@ export class ApplicationService {
     return ApiClient.Application.getApplicationData(applicationId);
   }
 
+  async getApplication(
+    applicationId: number,
```

Comment:
The option `loadDynamicData` for the query param is missing here.

Reply:
updated